azure/templates/post-deploy.yml

```diff
@@ -95,11 +97,19 @@ steps:
       export JWT_PRIVATE_KEY_ABSOLUTE_PATH="$(Pipeline.Workspace)/secrets/$(JWT_TESTING_PRIVATE_KEY)"
       export client_id="$(INTROSPECTION_CLIENT_ID)"
       export client_secret="$(INTROSPECTION_CLIENT_SECRET)"
+
       if [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]
       then
+       poetry run pytest -v -m "not nhsd_apim_authorization and not mock_provider_int" --api-name=gp-connect-appointments-management-fhir --proxy-name=$PROXY_NAME -o junit_logging=all --junitxml=test-report.xml
+
+      elif [[ $APIGEE_ENVIRONMENT == "int" ]]
+      then
+       export DEFAULT_CLIENT_ID="$(INT_CLIENT_ID)"
+       export DEFAULT_CLIENT_SECRET="$(INT_CLIENT_SECRET)"
```

Comment:
Everything else is fine. But you also need to explicitly export `APIGEE_APP_ID` here. Because for `int` `env`, it don't create app itself and will throw error

Reply:
8774d60